include/image-commands.mk

```diff
@@ -124,6 +124,7 @@ define Build/append-uImage-fakehdr
 		-s \
 		$@.fakehdr
 	cat $@.fakehdr >> $@
+	cat $@.fakehdr >> $(IMAGE_KERNEL).fakehdr
```

Comment:
Err, this changes behavior for _all_ devices using that recipe...
This should be made in a separate commit, and somebody should check the effect on the relevant devices ...

Reply:
All set; all changes to `include/image-commands.mk` are now dropped.